articles/hooks/extensibility-points/pre-user-registration.md

```diff
@@ -172,3 +172,28 @@ When we run this Hook, the response object is:
   }
 }
 ```
+
+## Sample script: Customize the error message and language for user messages
+
+In this example, we use a Hook to prevent a user from registering, then return a custom error message in our tenant logs and show a custom, translated error message to the user when they are denied. To use the translation functionality, your tenant must be configured to use the [Universal Login - New Experience](/universal-login/new).
```

Comment:
Which part of this is available only with Universal Login - New? Is it just the translation parameter, the ability to show a customized message to the user (regardless of language), or is it both?

Reply:
Ah good point.

Both the translation parameter AND the error message showing up in the signup form are only available in Universal Login - New.